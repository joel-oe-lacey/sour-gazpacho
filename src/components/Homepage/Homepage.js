import React from 'react';
import './Homepage.scss';


const Homepage = () => {
    return (
        <section className="homepage"></section>
    )
}

export default Homepage;