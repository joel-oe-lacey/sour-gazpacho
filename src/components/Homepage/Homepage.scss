@import '../../variables';

.homepage {
  height: 92%;
  width: 100%;
}

.movie-list-title {
  position: absolute;
  font-size: 50px;
  color: $primary-white-color;
}

.movie-homepage {
  display: flex;
  flex-flow: row wrap;
  justify-content: space-around;
  margin-top: 250px;

}

.movie-centerpiece {
  display: flex;
  flex-direction: row;
  justify-content: center;
  margin-bottom: 150px;
  margin-left: -30%;
}

.biscuits {
  width: 700px;
  margin: 50px;
  margin-left: 25%;
}
