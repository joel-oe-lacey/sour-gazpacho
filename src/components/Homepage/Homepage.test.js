import React from 'react';
import Homepage from './Homepage';
import { shallow } from 'enzyme'


describe('Homepage', () => {


  it('should match a snapshot as expected', () => {
    const allMovies = [{
            "id": 21,
            "title": "Sonic the Hedgehog",
            "poster_path": "https://image.tmdb.org/t/p/original//aQvJ5WPzZgYVDrxLX4R6cLJCEaQ.jpg",
            "backdrop_path": "https://image.tmdb.org/t/p/original//tCUcf3oNWMW8kwAj3WC6CvIN5ah.jpg",
            "release_date": "2020-02-12",
            "overview": "Based on the global blockbuster videogame franchise from Sega, Sonic the Hedgehog tells the story of the worldâ€™s speediest hedgehog as he embraces his new home on Earth. In this live-action adventure comedy, Sonic and his new best friend team up to defend the planet from the evil genius Dr. Robotnik and his plans for world domination.",
            "average_rating": 7
        },
        {
            "id": 22,
            "title": "Parasite",
            "poster_path": "https://image.tmdb.org/t/p/original//7IiTTgloJzvGI1TAYymCfbfl3vT.jpg",
            "backdrop_path": "https://image.tmdb.org/t/p/original//TU9NIjwzjoKPwQHoHshkFcQUCG.jpg",
            "release_date": "2019-05-30",
            "overview": "All unemployed, Ki-taek's family takes peculiar interest in the wealthy and glamorous Parks for their livelihood until they get entangled in an unexpected incident.",
            "average_rating": 4
        }]
    const wrapper = shallow(<Homepage allMovies={allMovies}/>)

    expect(wrapper).toMatchSnapshot()
  })
})
