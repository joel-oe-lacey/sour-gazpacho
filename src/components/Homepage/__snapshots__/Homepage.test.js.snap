// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage should match a snapshot as expected 1`] = `
<section
  className="homepage"
>
  <MovieCard
    averageRating={7}
    key="21"
    title="Sonic the Hedgehog"
  />
  <MovieCard
    averageRating={4}
    key="22"
    title="Parasite"
  />
</section>
`;
